docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3594,34 +3594,60 @@ opportunity for duplication. Since this mechanism occurs within the
-
-The user can then pick a rate of duplication between 0 and 100%. If the value selected
-is higher than the calculated maximum rate in the sim, a warning will be issued to users
-explaining that the selected rate is greater than the maximum available.
+Guardian-based duplication is applied to three mutually exclusive categories of
+simulants based on age and GQ status: Simulants younger than 18  (<18) and not
+in GQ; simulants 18-23 (18 <= age < 24) and not in GQ; and simulants under 24
+(<24) and in GQ.
+
+For each of the three categories of simulants, the maximum duplication rate will
+be calculated based on those who have a guardian living at a different address
+in the sim. Note that all simulants in *college* GQ are initialized with a
+guardian living at a different address, but this is not true for simulants in
+other types of GQ, so all three maximum duplication rates will be less than
+100%.
+
+The user can then pick a rate of duplication between 0 and 100% **for each of
+the three categories of simulants**. A default duplication rate of 5% will be
+selected for each of the three categories of simulants. That is, each simulant
+under 24 is duplicated at a guardian's address with default probability 0.05,
+and there should be three user parameters for overriding this probability, one
+for each simulant category. The duplicated row should have the same
+simulant-specific attributes as the original, such as name and birth date, but
+different address-specific attributes such as address fields and relation to
+reference person.
 
-A default value of 5% will be selected.
+If the user selects a duplication rate that is higher than the calculated
+maximum rate in the sim, a warning should be issued explaining that the selected
+rate is greater than the maximum available, and the actual rate of duplication
+should be set to the maximum for the specified simulant category. More
+precisely, the conditional probabiity that a row in the spceified category is
+duplicated, given that the row is eligible for guardian-based duplication,
+should be set to 1.
 
 .. note::
 
     If finding the maximum rate proves too difficult to implement, we can reassess this approach
 
-For college GQ simulants aged less than 24, all are assigned to a guardian who
-by definition lives at a different address. This means that theoretically
-the maximum noise level is 100% for this group, however, that would add
-significantly to the dataset and so it not allowed here. The default rate
-will be set to 5%, with a minimum of 0% and a maximum of 25%.
+To create guardian-based duplicates, each duplicated simulant will be included
+in the final dataset twice, once at their address of residence and once at their
+guardian's address. If a simulant has more than 1 guardian living at a different
+address, only duplicate them once, for a maximum of 2 occurences in the end
+dataset. Select the guardian at random.
```

Comment:
Most of this language is from the original version. @SylLutze @zmbc Did we restrict to two occurrences just for simplicity? I think I'd prefer to have the possibility of having more duplicates, but I guess it's not entirely clear how to determine when extra duplicates would happen based on the user parameters we've specified.

Reply:
I agree it would be good to have more duplicates possible, though we would be limited to 3 occurrences due to the 2-guardian system. The thing that seems tricky is how to design intuitive parameters that also allow that.